### ISSUE TITLE
unitypackage release

### ISSUE THREAD

**Issue by bsimser** [devdogio/Rucksack#2]
Would be nice to have a .unitypackage release file created. This could be manual or even automated against any code changes (if you're accepting PRs). Would be easier for people to use rather than having to download zips, dealing with submodules, etc.

**Comment by RErwandi** [comment]
Yeah, most project provide .unitypackage release file. I tried to install rucksack but getting error with general2 file

**Comment by bsimser** [comment]
The general2 (which is a modified version of the general library for Rucksack) is available on bitbucket here: https://bitbucket.org/jjahuijbregts/devdog.general2/src/master/

They just haven't updated the GitHub repo with it yet.
